### ISSUE TITLE
Various minor issues (Quests/NPCs)

### ISSUE THREAD

**Issue by ajgration** [ZhengPeiRu21/mod-individual-progression#182]
Undercity Guardian (5624) missing Inscription under Professions tab.
Quest Well Watcher Solanian (10068-10073) gave me 0xp on a mage, seems to be the same for all variants in the SQL. Should be 20xp as per retail.
Quest The Power to Destroy (5725) objective tracker reading "Return to Varimathras at Lordamere Lake in Silverpine Forest" instead of Undercity.
Quest Report to Kadrak (6541, 6542), should be able to accept and complete both the Stonetalon and Barrens versions of this quest, so long as you don't pick up The Warsong Reports (6543). Currently only able to complete one or the other (as per https://www.wowhead.com/wotlk/quest=6542/report-to-kadrak#comments). (assuming this is an AC issue)
I got this console error questing in barrens "SmartScript::ProcessAction: Entry 14908 SourceType 0, Event 7, Link Event 8 not found or invalid, skipped." (assuming this is an AC issue).

**Edit 19/09**
Iridescent Hammer (5541) has wrong weapon speed. Should have 1.80 speed instead of 2.80.

**Edit 24/09**
Journeyman profession trainers not teaching anything. All their recipes are learned from Artisan trainers (tested in undead/blood elf zones).
Warlocks Summon Felsteed is on the trainer (undead zones, not sure about others), should instead only be a quest reward from the quest Summon Felsteed (4490).

**Comment by ZhengPeiRu21** [comment]
Thank you for the reports. I'll look into these issues. Regarding the Undercity guardian missing Inscription, I unfortunately don't have a source for their gossip for that option - it only existed between WotLK launch and the Wrathgate patch when they were removed. If you happen to come across the correct dialog, please let me know. Otherwise, I will write an appropriate guess.

**Comment by ajgration** [comment]
They didn't completely remove the guards in 3.3, some remained in the sewers. I found the solution in the AC gossip_menu_option.sql 

INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (2847, 8, 0, 'Inscription', 48811, 1, 1, 10019, 344, 0, 0, NULL, 0, NULL);

**Comment by Grimfeather** [comment]
I fixed some of these issues:
- missing Undercity guards gossip menu for inscription
- removed Summon Felsteed spell from warlock trainers
- updated quest text for Power to Destroy
- updated attack speed for Iridescent Hammer

The xp issue and also reputation btw, for Quest Well Watcher Solanian is an AzerothCore issue.
The quest is still available in WotLK.
I did look into it for a little bit, but it seems altering QuestXP.dbc is involved.

I'll have a look at the journeyman trainers still.
But I predict this is a lot of work, so I'll do this separately.
